designer/client/src/components/toolbars/search/utils.ts

```diff
@@ -92,18 +109,92 @@ export function useFilteredNodes(filterValues: string[]): {
         () =>
             allNodes
                 .map((node) => {
-                    const edges = allEdges
-                        .filter((e) => e.from === node.id)
-                        .filter((e) => matchFilters(e.edgeType?.condition, filterValues));
+                    let edges: Edge[] = [];
+                    let groups: string[] = [];
+
+                    if (isSimpleSearch) {
+                        edges = allEdges
+                            .filter((e) => e.from === node.id)
+                            .filter((e) => matchFilters(e.edgeType?.condition, [searchQuery.plainQuery]));
+
+                        groups = findFields([searchQuery.plainQuery], node)
+                            .concat(edges.length ? "edgeExpression" : null)
+                            .map((name) => displayNames[name])
+                            .filter(Boolean);
+
+                        return { node, edges, groups };
+                    } else {
+                        const edgesAux = allEdges
```

Comment:
I didn't analyze this logic, but somewhere must be the problem, i tried to search just by the advanced search but seems like it doesn't work as expected


https://github.com/user-attachments/assets/a9223bb1-4a6e-40ab-bed9-39922f2678c7

Reply:
Yeah, that was a bug. Fixed